tests/snapshots/TestCLICommands_atmos_vendor_pull_ssh.stderr.golden

```diff
@@ -0,0 +1,11 @@
+DEBU Using command line argument '--logs-level=Debug'
+DEBU processStoreConfig atmosConfig.StoresConfig=map[]
+DEBU processStoreConfig atmosConfig.StoresConfig=map[]
+DEBU processStoreConfig atmosConfig.StoresConfig=map[]
+INFO Vendoring from 'vendor.yaml'
+WARN No TTY detected. Falling back to basic output. This can happen when no terminal is attached or when commands are pipelined.
+INFO ✓ terrafrom-null-label (0.25.0)
```

Comment:
I was expecting to see some debug log messages from custom detector. Looks like our dry run is too dry.